### ISSUE TITLE
Error I got after fresh install from git on Ubutuntu 14.04 with Python 2.7.6

### ISSUE THREAD

**Issue by cristalix** [Bitmessage/PyBitmessage#942]
The error I got after fresh install from git on Ubutuntu 14.04 with Python 2.7.6 (default, Oct 26 2016, 20:32:47)

```
name@comp:~$ ~/PyBitmessage/src/bitmessagemain.py 
Loading existing config files from /home/name/.config/PyBitmessage/
Failed to load debug config from /home/name/.config/PyBitmessage/logging.dat, using default logging config
(<class 'ConfigParser.NoSectionError'>, No section: 'formatters', <traceback object at 0xb69a13c4>)
2017-01-30 07:58:03,524 - CRITICAL - PyBitmessage requires PyQt unless you want to run it as a daemon and interact with it using the API. You can download it from http://www.riverbankcomputing.com/software/pyqt/download or by searching Google for 'PyQt Download' (without quotes).
Traceback (most recent call last):
  File "/home/name/PyBitmessage/src/bitmessageqt/__init__.py", line 14, in <module>
    from PyQt4 import QtCore, QtGui
ImportError: No module named PyQt4
2017-01-30 07:58:03,531 - CRITICAL -   File "/home/name/PyBitmessage/src/bitmessagemain.py", line 301, in <module>
    mainprogram.start(shared.safeConfigGetBoolean('bitmessagesettings', 'daemon'))
  File "/home/name/PyBitmessage/src/bitmessagemain.py", line 165, in start
    shared.thisapp = singleton.singleinstance("", daemon)
  File "/home/name/PyBitmessage/src/singleton.py", line 31, in __init__
    import bitmessageqt
  File "/home/name/PyBitmessage/src/bitmessageqt/__init__.py", line 22, in <module>
    sys.exit()

2017-01-30 07:58:03,531 - CRITICAL - <type 'exceptions.NameError'>: name 'sys' is not defined
```

What do I miss?

Should I install some PyQt? How to install this staff? Where to place the binaries and how to say PyBitmessage that it is on its place? Shit! I already can see killed 2 - 4 hours for trying to install this program. Simplify the process!

**Comment by cristalix** [comment]
I couldn't install `python-qt4` with `sudo apt-get install python-qt4`. I get the error "`Unable to locate package`". This solved the issue: http://askubuntu.com/a/844032

**Comment by cristalix** [comment]
How to install bitmessage on Ubuntu https://scottlinux.com/2013/07/26/bitmessage-how-to-for-debian-and-ubuntu-linux/

**Comment by cristalix** [comment]
I now I got "ERROR - HTTP connection error". It cannot connect. Fucking piece of shit.

**Comment by PeterSurda** [comment]
That error message is about namecoin, it's misleading and fixed in the v0.6 branch.
